.github/workflows/Helix-PR-CI.yml

```diff
@@ -1,7 +1,7 @@
 name: Helix PR CI
 on:
   pull_request:
-    branches: [ master ]
+    branches: [ master, metaclient ] # TODO: remove side branch
```

Comment:
should you remove this??

Reply:
Will remove in future commit. Will keep the merge PR pure for merge.